### PR TITLE
Icon: Rename 'link' Icon to 'visit' + codemod, add new 'link' Icon, update Docs

### PR DESCRIPTION
**Pull request by AlbertCarreras**
Renamedrenamed 'link' icon to 'visit' + codemod, add new 'link' icon, updated Docs #1406

Update Dash icon

BEFORE | AFTER
------------ | -------------
image | image
image | image

## Codemod

 Converts
```
 <Icon icon='link'/> to <Icon icon='visit'/>
 <IconButton icon='link'/> to <IconButton icon='visit'/>
 <Pog icon='link'/> to <Pog icon='visit'/>
 <Button iconEnd='link'/> to <Button iconEnd='visit'/>
```
 
 Console logs each Module with 'items' prop for a manual review.

Run ` yarn codemod --parser=flow -t=packages/gestalt-codemods/19.0.0/rename-link-icon-to-visit.js relative/path/to/your/code`

### CODE REVIEW

File: packages/gestalt/src/icons/visit.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M21 15.05c1.1 0 2 .9 2 2V21c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h4a2 2 0 1 1 0 4H5v14h14v-1.95c0-1.1.9-2 2-2zM12 14c-.51 0-1.02-.2-1.41-.59-.79-.78-.79-2.04 0-2.82L16.16 5 14 5.01c-1.1 0-2-.9-2-2 0-1.11.89-2 2-2L20.99 1h2v2l.01 7a2 2 0 1 1-4 0V7.83l-5.59 5.58c-.39.39-.9.59-1.41.59z"/></svg>
```
Comment (AlbertCarreras) [inline packages/gestalt/src/icons/visit.svg:1]: Previously link.js

File: packages/gestalt/src/icons/link.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M21 15.05c1.1 0 2 .9 2 2V21c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h4a2 2 0 1 1 0 4H5v14h14v-1.95c0-1.1.9-2 2-2zM12 14c-.51 0-1.02-.2-1.41-.59-.79-.78-.79-2.04 0-2.82L16.16 5 14 5.01c-1.1 0-2-.9-2-2 0-1.11.89-2 2-2L20.99 1h2v2l.01 7a2 2 0 1 1-4 0V7.83l-5.59 5.58c-.39.39-.9.59-1.41.59z"/></svg>
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M21 7.24l-4.05 4.05-1.06-1.06.67-.67a1.5 1.5 0 1 0-2.12-2.12l-.67.67-1.06-1.06L16.76 3zm-9.7 9.7L7.23 21 3 16.76l4.05-4.05 1.06 1.06-.67.67a1.5 1.5 0 0 0 2.12 2.12l.67-.67zM14.63.89l-4.05 4.05a3 3 0 0 0 0 4.24l1.06 1.06-1.42 1.42-1.06-1.06a3 3 0 0 0-4.24 0L.88 14.64a3 3 0 0 0 0 4.24l4.24 4.24a3 3 0 0 0 4.24 0l4.05-4.05a3 3 0 0 0 0-4.24l-1.06-1.06 1.42-1.42 1.06 1.06a3 3 0 0 0 4.24 0l4.05-4.05a3 3 0 0 0 0-4.24L18.88.88a3 3 0 0 0-4.24 0z"/></svg>
```
Comment (AlbertCarreras) [inline packages/gestalt/src/icons/link.svg:1]: new icon: link.svg

**Review by anniemochi — APPROVED** [review]

File: docs/src/components/Card.js [hunk 3]
```diff
@@ -60,11 +60,8 @@ export default function Card({
             <Heading size={headingSize}>{name}</Heading>
             <Tooltip inline text="Copy link">
               <IconButton
-                dangerouslySetSvgPath={{
-                  __path:
-                    'M21.001 7.241l-4.053 4.052-1.06-1.06.672-.672a1.5 1.5 0 10-2.121-2.121l-.671.672-1.061-1.06L16.759 3l4.242 4.241zm-9.708 9.708l-4.052 4.052-4.242-4.241 4.053-4.053 1.059 1.06-.671.672a1.5 1.5 0 002.121 2.121l.671-.672 1.061 1.061zM14.639.879l-4.053 4.052a3 3 0 000 4.242l1.061 1.06-1.415 1.414-1.06-1.061a3 3 0 00-4.241 0L.879 14.638a2.998 2.998 0 000 4.242l4.241 4.242a3 3 0 004.241 0l4.053-4.052a3 3 0 000-4.242l-1.06-1.061 1.414-1.413 1.06 1.06a3 3 0 004.241 0l4.052-4.052a2.998 2.998 0 000-4.242L18.88.879a2.997 2.997 0 00-4.241 0z',
-                }}
                 accessibilityLabel={`Copy link to ${name}`}
+                icon="link"
```
Comment (ayeshakmaz) [inline docs/src/components/Card.js:64]: 🎉

**Review by ayeshakmaz — APPROVED** [review]
LGTM 🎉

**Pull request merged**